parm/config/gfs/config.resources

```diff
@@ -623,15 +623,15 @@ case ${step} in
 
     case "${CASE}" in
       "C48" | "C96" | "C192")
-        declare -x "wtime_${step}"="00:30:00"
+        declare -x "wtime_${step}"="00:15:00"
         declare -x "wtime_${step}_gfs"="03:00:00"
         ;;
       "C384")
-        declare -x "wtime_${step}"="00:20:00"
+        declare -x "wtime_${step}"="00:10:00"
```

Comment:
Interesting that the C384 `wtime` is less than the lower resolutions `wtime`. The enkfgdasefcs jobs in recent tests on WCOSS2 at C96C48L127 ran ~9.5mins for two members. Don't need to change in this PR but some testing to get better timings would be good, especially if we're going to have up to 80 jobs (1/member) now.

Reply:
I just halved all the walltimes since one forecast will be run instead of two. It wasn't validated.